### ISSUE TITLE
Feature request: choose a folder to download

### ISSUE THREAD

**Issue by filiptrplan** [civitai/sd_civitai_extension#77]
It would be great that when you click the Add to SD button on the website, it would give you a dropdown with all your local folders inside, for example the lora folder and let you choose where to download.

**Comment by Aptronymist** [comment]
Yes, please! I keep mine in subfolders, at the worst in my Unsorted folder under each network, and this just makes a mess I have to then clean up.
